builtin/logical/ssh/path_sign.go

```diff
@@ -28,7 +28,8 @@ type creationBundle struct {
 	ValidPrincipals []string
 	PublicKey       ssh.PublicKey
 	CertificateType uint32
-	TTL             time.Duration
+	ValidBefore     uint64
+	ValidAfter      uint64
```

Comment:
replaced `TTL` here with `ValidAfter` and `ValidBefore`, the logic to calculate them was pulled up into the sign function.

this is done because the sign function holds the information necessary to decide if `ValidAfter` and `ValidBefore` should be calculated based on the `ttl` (specified or derived), or, if they are to be given explicitly by the new `valid_after` and `valid_before` parameters